booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java

```diff
@@ -105,6 +105,8 @@ private String generateSchemaValidVesselIMONumber() {
   private void supplyScenarioParameters(JsonNode actionPrompt) {
     log.info("Carrier.supplyScenarioParameters(%s)".formatted(actionPrompt.toPrettyString()));
     var scenarioType = ScenarioType.valueOf(actionPrompt.required("scenarioType").asText());
+    var assignCbr = actionPrompt.required("assignCbr");
+    persistentMap.save("assignCbr",assignCbr);
```

Comment:
Why is `assignCbr` both in the action, stored into the persistent map AND in the DSP?  The persistent map appears weird and is probably not good if we ever support threading.